js/modules/k6/ws/ws_test.go

```diff
@@ -1288,3 +1291,48 @@ func TestCookieJar(t *testing.T) {
 
 	assertSessionMetricsEmitted(t, metrics.GetBufferedSamples(ts.samples), "", sr("WSBIN_URL/ws-echo-someheader"), statusProtocolSwitch, "")
 }
+
+func TestWSConnectWithThrowErrorOption(t *testing.T) {
+	t.Parallel()
+
+	logHook := &testutils.SimpleLogrusHook{HookedLevels: []logrus.Level{logrus.WarnLevel}}
+	testLog := logrus.New()
+	testLog.AddHook(logHook)
+	testLog.SetOutput(io.Discard)
+
+	t.Run("ThrowEnabled", func(t *testing.T) {
+		t.Parallel()
+		ts := newTestState(t)
+		ts.state.Logger = testLog
```

Comment:
I don't think this is a problem - but this is still shared between the two and you seem to *not* use it. So maybe it shouldn't be set at all :shrug:

Reply:
i only set `Logger` with throw option is false.

Reply:
if without `Logger`, when throw option is false, `ws` can not log warn message